### DISCUSSION TITLE
Kazakh Language

### DISCUSSION THREAD

**Discussion by yeldarx** [ggml-org/whisper.cpp#727] [Ideas]
Hi! Could you add Kazakh to the list of languages?

**Comment by indokid9999** [comment]
Theres a code for "kazakh" in the file whisper.cpp.It is defined as "kk"".
Maybe try adding "-l kk" to the command line?
try the followingin your command line and use backslash  if using windows.

./main -m ./models/ggml-base.bin -f  kazakh_audio.wav -l kk
